services/user.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+
+    user = get_user_model().objects.create_user(username=username,
+                                                password=password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
```

Comment:
You call the `save()` method twice in the `create_user` function. The first time is in the `create_user` method of the User model, the second time is on line 22. You should avoid calling `save()` multiple times. You can pass `email`, `first_name`, and `last_name` as parameters to the `create_user` method to avoid the second `save()` call. This will improve performance.